WORKSPACE

```diff
@@ -17,56 +17,41 @@
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-# proto_library rules implicitly depend on @com_google_protobuf//:protoc,
-# which is the proto-compiler.
-# cc_proto_library rules implicitly depend on @com_google_protobuf_cc//:cc_toolchain,
-# which is the C++ proto runtime (base classes and common utilities).
-# This statement defines the @com_google_protobuf repo.
-git_repository(
-    name = "com_google_protobuf",
-    remote = "https://github.com/google/protobuf.git",
-    tag = "v3.6.1.3",
-)
-
 http_archive(
     name = "com_github_glog_glog",
-    build_file = "@//glog_files:BUILD",
-    urls = ["https://github.com/google/glog/archive/v0.3.5.tar.gz"],
-    strip_prefix = "glog-0.3.5",
+    sha256 = "f28359aeba12f30d73d9e4711ef356dc842886968112162bc73002645139c39c",
+    strip_prefix = "glog-0.4.0",
+    urls = ["https://github.com/google/glog/archive/v0.4.0.tar.gz"],
 )
 
 # gflags
 # Needed for glog
-git_repository(
-    name = "com_github_gflags_gflags",
-    remote = "https://github.com/gflags/gflags.git",
-    tag = "v2.2.2",
-)
-
-# BoringSSL
 http_archive(
-    name = "boringssl",
+    name = "com_github_gflags_gflags",
+    sha256 = "34af2f15cf7367513b352bdcd2493ab14ce43692d2dcd9dfc499492966c64dcf",
+    strip_prefix = "gflags-2.2.2",
     urls = [
-        "https://mirror.bazel.build/github.com/google/boringssl/archive/a0fb951d2a26a8ee746b52f3ba81ab011a0af778.tar.gz",
-        "https://github.com/google/boringssl/archive/a0fb951d2a26a8ee746b52f3ba81ab011a0af778.tar.gz",
+        "https://mirror.bazel.build/github.com/gflags/gflags/archive/v2.2.2.tar.gz",
+        "https://github.com/gflags/gflags/archive/v2.2.2.tar.gz",
     ],
-    sha256 = "524ba98a56300149696481b4cb9ddebd0c7b7ac9b9f6edee81da2d2d7e5d2bb3",
-    strip_prefix = "boringssl-a0fb951d2a26a8ee746b52f3ba81ab011a0af778",
-)
-
-# Abseil C++ libraries
-git_repository(
-    name = "com_google_absl",
-    remote = "https://github.com/abseil/abseil-cpp.git",
-    tag = "20181200",
 )
 
 # gRPC
-git_repository(
+http_archive(
     name = "com_github_grpc_grpc",
-    remote = "https://github.com/grpc/grpc.git",
-    tag = "v1.19.0",
+    sha256 = "c2ab8a42a0d673c1acb596d276055adcc074c1116e427f118415da3e79e52969",
+    strip_prefix = "grpc-1.27.3",
+    urls = [
+        "https://github.com/grpc/grpc/archive/v1.27.3.tar.gz",
+    ],
 )
 
-load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
+load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps", "grpc_test_only_deps")
+
 grpc_deps()
+
+grpc_test_only_deps()
```

Comment:
Could you add a comment what this is needed for?

Reply:
I realized it's not needed at all and removed it.